Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -346,7 +356,7 @@ public static function initializeCacheManagement(Bootstrap $bootstrap)
         $environment = $bootstrap->getEarlyInstance(Environment::class);
 
         $cacheFactoryObjectConfiguration = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_OBJECTS, CacheFactoryInterface::class);
-        $cacheFactoryClass = isset($cacheFactoryObjectConfiguration['className']) ? $cacheFactoryObjectConfiguration['className'] : CacheFactory::class;
+        $cacheFactoryClass = $cacheFactoryObjectConfiguration && isset($cacheFactoryObjectConfiguration['className']) ? $cacheFactoryObjectConfiguration['className'] : CacheFactory::class;
```

Comment:
That additional condition is not necessary in PHP with `isset()` ;) But no blocker

Reply:
Nice, didn't know that. I've removed the condition.